### ISSUE TITLE
checks failing now, because of changeing stops to aborts

### ISSUE THREAD

**Issue by fhindenlang** [hopr-framework/hopr#56]
Hi guys, 
in the new FEM feature branch, #51 , we have changed all  stops to aborts. Now before, failing jobs were marked successful, because the stop was not found by the cmake tests. 
I think this change needs to  be addressed in the latest release 1.2.1, before continuing in the feature branch...

**Comment by kopperp** [comment]
I think that is actually an issue introduced by me. We have constant memory leaks from the readintools and the fix is currently crashing after successfully creating the mesh. I will check and push a fix.

**Comment by fhindenlang** [comment]
yeah lets close this issue, we just fix everything in the FEM feature branch, and will then have a new release.
